clicommand/global.go

```diff
@@ -202,6 +214,53 @@ func apiFlags() []cli.Flag {
 	}
 }
 
+func cacheFlags() []cli.Flag {
+	return []cli.Flag{
+		cli.StringFlag{
```

Comment:
A `cli.StringSliceFlag` might be a better fit.

Reply:
done.